src/test/java/com/debatetimer/controller/member/MemberControllerTest.java

```diff
@@ -0,0 +1,57 @@
+package com.debatetimer.controller.member;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.debatetimer.BaseControllerTest;
+import com.debatetimer.controller.member.dto.MemberCreateRequest;
+import com.debatetimer.controller.member.dto.MemberCreateResponse;
+import com.debatetimer.controller.member.dto.TableResponses;
+import com.debatetimer.domain.member.Member;
+import com.debatetimer.domain.parliamentary_debate.ParliamentaryTable;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class MemberControllerTest extends BaseControllerTest {
+
+    @Nested
+    class CreateMember {
+
+        @Test
+        void 회원을_생성한다() {
+            MemberCreateRequest request = new MemberCreateRequest("커찬");
+
+            MemberCreateResponse response = RestAssured.given().log().all()
+                    .contentType(ContentType.JSON)
+                    .body(request)
+                    .when().post("/api/member")
+                    .then().log().all()
+                    .statusCode(201)
+                    .extract().as(MemberCreateResponse.class);
+
+            assertThat(response.nickname()).isEqualTo("커찬");
```

Comment:
[사소한 제안]

response의 nickname이 request의 nickname과 동일하다는 점에서 문자열로 하드코딩하기보다 request에서 꺼내서 검증해주는 것도 좋을 것 같아요. 반영은 커찬의 의사에 맡길게요 ㅎㅎ

```suggestion
            assertThat(response.nickname()).isEqualTo(request.nickname());
```

Reply:
반영하겠습니다~